nrf_modem/doc/ug_nrf_modem_porting_os.rst

```diff
@@ -134,10 +134,10 @@ nrf_modem_os_trace_put
 
 This function puts the trace string to the desired medium, typically UART.
 However, the medium used to forward and store the traces is up to the implementation and must be initialized correctly before using.
-Once the traces are processed/stored, the :c:func:`nrf_modem_trace_processed_callback` must be called.
-If you are not interested in traces you only need to ensure that :c:func:`nrf_modem_trace_processed_callback` is called for each received trace.
-Until the :c:func:`nrf_modem_trace_processed_callback` is called, the Modem library will not free up the memory allocated for that trace in the trace memory area.
-Since the modem uses this trace memory area to send traces, not calling :c:func:`nrf_modem_trace_processed_callback` may lead to lost modem traces.
+Once the traces are processed or stored, the :c:func:`nrf_modem_trace_processed_callback` must be called.
+If you want the traces, you need to ensure that :c:func:`nrf_modem_trace_processed_callback` is called for each received trace.
```

Comment:
We should rephrase this, it lost its meaning like this.
The idea is: regardless of whether the application is interested in processing the trace data, the application must call `nrf_modem_trace_processed_callback` for each received trace.

Reply:
fixed.